ultravox/tools/ds_tool.py

```diff
@@ -112,6 +136,8 @@ def main(args: DatasetToolArgs):
 
     for split, ds_split in data_dict.items():
         print(f'Processing split "{split}"...')
+        if args.shuffle:
```

Comment:
can't we shuffle the dataset once it's created (ie in training?)

Reply:
Agree that shuffle without `num_samples` doesn't make much sense, but for SODA we don't want to convert the whole set.